README.md

```diff
@@ -12,13 +12,20 @@
 
 - [Release Notes](./widget/release_notes.md)
 
-**Note**: Preview releases are a beta release. This means that release is intended to get feedback and might not be best suited for production level deployments. The functionality might change in backward-incompatible ways or have limited support. A beta release is not subject to any SLA, Quality Assurance or deprecation policy. Feature availability and support for preview releases will continue to improve as the solution/feature matures.
+
+<table>
+    <td><strong>NOTE</strong></td><td>Preview releases are a beta release, is intended to get feedback and might not be best suited for production level deployments. The functionality might change in backward-incompatible ways or have limited support. A beta release is not subject to any SLA, Quality Assurance or deprecation policy. Feature availability and support for preview releases will continue to improve as the solution/feature matures.</td>
```

Comment:
If we happen to remove preview tag then, we will need to remove this text as well